projects/packages/my-jetpack/_inc/components/product-cards-section/protect-card/auto-firewall-status.tsx

```diff
@@ -0,0 +1,103 @@
+import { __ } from '@wordpress/i18n';
+import useProduct from '../../../data/products/use-product';
+import { getMyJetpackWindowInitialState } from '../../../data/utils/get-my-jetpack-window-state';
+import useMyJetpackConnection from '../../../hooks/use-my-jetpack-connection';
+import ShieldInactive from './assets/shield-inactive.svg';
+import ShieldOff from './assets/shield-off.svg';
+import ShieldSuccess from './assets/shield-success.svg';
+import { InfoTooltip } from './info-tooltip';
+import { useProtectTooltipCopy } from './use-protect-tooltip-copy';
+import type { ReactElement, PropsWithChildren } from 'react';
+
+export const AutoFirewallStatus = () => {
+	const slug = 'protect';
+	const { detail } = useProduct( slug );
+	const { isPluginActive = false } = detail || {};
+	const { isSiteConnected } = useMyJetpackConnection();
+	const {
+		protect: { wafConfig: wafData },
+	} = getMyJetpackWindowInitialState();
+	const { jetpack_waf_automatic_rules: isAutoFirewallEnabled } = wafData;
+
+	if ( isPluginActive && isSiteConnected ) {
+		if ( isAutoFirewallEnabled ) {
+			return <WafStatus status="success" />;
+		}
+
+		return <WafStatus status="inactive" />;
+	}
+
+	return <WafStatus status="off" />;
+};
+
+/**
+ * WafStatus component
+ *
+ * @param {PropsWithChildren} props - The component props
+ * @param {'success' | 'inactive' | 'off'} props.status - The number of threats
+ *
+ * @returns {ReactElement} rendered component
+ */
+function WafStatus( { status }: { status: 'success' | 'inactive' | 'off' } ) {
+	const slug = 'protect';
+	const { detail } = useProduct( slug );
+	const { hasPaidPlanForProduct: hasProtectPaidPlan } = detail;
+	const tooltipContent = useProtectTooltipCopy();
+	const { autoFirewallTooltip } = tooltipContent;
+
+	if ( status === 'success' ) {
+		return (
+			<>
+				<div>
+					<img
+						className="value-section__status-icon"
+						src={ ShieldSuccess }
+						alt={ __( 'Shield icon - Auto-Firewall Status: On', 'jetpack-my-jetpack' ) }
+					/>
+				</div>
+				<div className="value-section__status-text">{ __( 'On', 'jetpack-my-jetpack' ) }</div>
+			</>
+		);
+	}
+	if ( status === 'inactive' ) {
+		return (
+			<>
+				<div>
+					<img
+						className="value-section__status-icon"
+						src={ ShieldInactive }
+						alt={ __( 'Shield icon - Auto-Firewall Status: Inactive', 'jetpack-my-jetpack' ) }
+					/>
+				</div>
+				<div className="value-section__status-text">{ __( 'Inactive', 'jetpack-my-jetpack' ) }</div>
+				{ ! hasProtectPaidPlan && (
```

Comment:
I wonder if we update the tooltip text if they have a plan to be something like "You have firewall disabled, visit your Protect settings to activate" and there could be a link to the settings page too 🤔  Idk not required, I thought it might be good. If the user sees this and doesn't know why it's inactive or how to activate it, that could help with retention and knowing how to use the product effectively

Reply:
Thanks @CodeyGuyDylan, Yeah I think that is a good idea! I actually was thinking the exact same thing!
I think I'll add it in the next, final PR, because that is when I'm going to ping Design to take a look at everything anyway, so I'll confirm/ask about the additional tooltip text at that time too. 👍